docs/start_with_docker_plus_virtenv.md

```diff
@@ -0,0 +1,258 @@
+c29b29edc871: Download complete                                     
+75fa503deb0b: Download complete                                     
+03297d3829eb: Download complete 
+fcf26cd86178: Download complete 
+5bcaaf1fd219: Download complete 
+431ffe29be39: Download complete 
+79e926b74f85: Download complete 
+4f4fb700ef54: Verifying Checksum 
+27ae57810c0a: Downloading [=>                                                 ]  19.43MB/570.5MB
+9ecd20cd6844: Download complete 
+51c071dfcd29: Download complete 
+438302fc8bd8: Download complete 
+44999d133959: Downloading [>                                                  ]  10.79MB/10.57GB
+a5ed971e796e: Pulling fs layer                                      
+f570a0dd636d: Waiting                                               
+1a08cbb00ee1: Waiting                                               
+```
+
+The Docker image can be large (i.e., > 10GB), so it takes time to download. You can plug an ethernet cable into your router and Stretch to speed up the download. You will only need to download each version of the *stretch-ai* Docker image a single time.
+
+After downloading the Docker image, the server will begin running on your robot. Your robot should beep twice and its lidar should start spinning. The terminal should display output from the robot's *stretch-ai* server like the following:
+
+```bash
+[server-12] j='joint_wrist_yaw' idx=10 idx_q=10
+[server-12] j='joint_wrist_pitch' idx=11 idx_q=11
+[server-12] j='joint_wrist_roll' idx=12 idx_q=12
+[server-12] ==========================================
+[server-12] Starting up threads:                                    
+[server-12]  - Starting send thread
+[server-12]  - Starting recv thread
+[server-12]  - Sending state information
+[server-12]  - Sending servo information
+[server-12] Running all...                                          
+[server-12] Starting to send full state
+[stretch_driver-3] [INFO] [1727454898.725969113] [stretch_driver]: Changed to mode = position
+```
+
+## Create a virtual env on your GPU Computer
+
+First, check to see if mamba is installed on your computer:
+
+```bash
+mamba
+```
+
+If you get an "command not found" error, then follow the install instructions for mamba here: https://github.com/conda-forge/miniforge#download
+
+Then, run:
+
+```bash
+# Install Git LFS - needed for large files like images
+sudo apt-get install git-lfs
+git lfs install
+
+# Clone the repository
+# Do not forget the --recursive flag to clone submodules
+git clone git@github.com:hello-robot/stretch_ai.git --recursive
+
+# Install system dependencies
+# Pyaudio will fail if these are not installed
+sudo apt-get install libasound-dev portaudio19-dev libportaudio2 libportaudiocpp0 espeak ffmpeg
+
+# Run install script to create a conda environment and install dependencies
+./install.sh
+```
+
+At the prompt to install Detic, say **no**. Take note of the name of the environment. It will be something like `stretch_ai_<version>`.
```

Comment:
How are you running this if you did not install detic?

Reply:
It's installed manually in a later step. Detic in the script fails to install for me. See issue https://github.com/hello-robot/stretch_ai/issues/177

Reply:
Why not just fix the script? Or delete it from the script?

Reply:
I see no harm to leaving it in the script so it can be fixed later.